app/api/routers/ask_gpt.py

```diff
@@ -2,21 +2,28 @@
 import openai
 from fastapi import APIRouter
 from dotenv import load_dotenv
+from pydantic import BaseModel
 from app.utils.anonymize import anonymize_text
 
 load_dotenv()
 ask_router = APIRouter()
 openai.api_key = os.getenv("OPENAI_API_KEY")
 
+class AskRequest(BaseModel):
```

Comment:
you could move this to a separate file named `models.py` in your folder `app/api/models/` (that was empty so far)

Reply:
if you don't mind sir I prefer deleting the models file since we are not using databases yet and keep this here cuz is not a database schema and used only by /ask endpoint